### PR TITLE
Update dockerfile to speedup the dockerbuild

### PR DESCRIPTION
**Pull request by c4software**
Hi,

The current Dockerfile doesn't use correctly the Docker cache system (when rebuild) I have modified the file to make it fast as hell ;).

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,11 +6,14 @@ RUN apt-get update && apt-get install -y redis-server
 # Install Mercurius
 RUN mkdir /src
 WORKDIR /src
-ADD . /src
+COPY package.json /src
 RUN npm install
+RUN npm build
```
Comment (marco-c) [inline Dockerfile]: Why is this needed?

Comment (c4software) [inline Dockerfile]: It's a mistake... In my latest commit I have removed it.
